packages/backend/migrations/20221012013413_create_grants_table.js

```diff
@@ -6,6 +6,7 @@ exports.up = function (knex) {
     table.string('continueId').nullable()
     table.string('continueToken').nullable()
     table.string('accessToken').nullable().unique()
+    table.string('managementUrl').nullable()
```

Comment:
Can we make this nonnullable, since `access_token.manage` is always defined?

Reply:
I think the same goes for `accessToken`